rosetta-source/src/main/rosetta/event-common-type.rosetta

```diff
@@ -571,13 +571,12 @@ type TransferBase:
     identifier Identifier (0..*) <"Represents a unique reference to the transfer.">
         [metadata scheme]
     quantity NonNegativeQuantity (1..1) <"Represents the amount of the asset to be transferred.">
-    observable Observable (0..1) <"Represents the object that is subject to the transfer, it could be an asset or a reference.">
+    asset Asset (1..1) <"Represents the object that is subject to the transfer, it could be an asset or a reference.">
     payerReceiver PartyReferencePayerReceiver (1..1) <"Represents the parties to the transfer and their role.">
     settlementDate AdjustableOrAdjustedOrRelativeDate (1..1) <"Represents the date on which the transfer to due.">
 
     condition FinancialUnitExists:
-        if observable exists
-        then quantity -> unit -> financialUnit exists
+        quantity -> unit -> financialUnit exists
```

Comment:
This condition needs to be rewritten now that the transfer's underlier is `Asset` not `Observable`:
* if asset is Cash, the unit must be currency.
* if asset is Instrument, must be financialUnit
* If asset is Commodity, must be capacity

Reply:
Done